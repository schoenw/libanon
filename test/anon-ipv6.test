#!/bin/bash
#
# Shell script for regression testing libanon (anon-ipv6).
#
# $Id$
#

ANON=../src/anon
PASSPHRASE=testing

RC=0
for file in anon-ipv6.*.in; do
    $ANON ipv6 -p $PASSPHRASE $file \
	| diff -u `basename $file .in`.out -
    if [ $? -ne 0 ]; then
 	RC=1
    fi
done

exit ${RC}