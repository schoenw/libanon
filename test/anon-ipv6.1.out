cc29:fb1a:b9fd:613c:e68:a03b:1efd:961f
cc29:fb1a:b9fd:613c:e68:a03b:1efd:961f
a3cf:3bda:5768:b87b:ac8c:cc56:a43:5235
dcce:77ba:d5ad:16df:c119:68b2:2f0a:2518
f7a3:94a6:da41:21f1:8fa6:bbe9:aa59:c001
fdf2:1602:ce3f:5a8a:5da9:597:e43b:1bce
eecc:4a6d:2f3d:775b:cb88:ae6f:a8a2:6fc1
e7bd:c302:a5f5:d85e:3308:768:2570:fa5f
e1ad:da60:80b0:bc67:c2c:7b50:3873:e40f
e527:7fea:c099:b4df:4e9d:73ac:2d3f:eb06
fefb:a377:29d:3a2d:e8a6:eb59:a391:d621
cc29:fb1a:b9fd:613c:e68:a03b:1efd:961f
cc29:fb1a:b9fd:613c:e68:a03b:1efd:961f
fefb:a377:29d:3a2d:e8a6:eb59:a391:d621
cc29:fb1a:b9fd:613c:e68:a03b:1efd:961f
cc29:fb1a:b9fd:613c:e68:a03b:1efd:961f
1:ce05:ee9:971:7f1f:1f8:ec7:c05b
1:ce05:ee9:971:7f1f:1f8:ec7:c05f
0:e9f8:3928:f76e:3f:3fc6:135:ffa6
0:e9f8:3928:f76e:3f:3fc6:135:ffa5
d0f1:f603:8e9:98c:3f5f:1f9:ffcf:bf9f
d0f1:f603:8e9:98c:3f5f:1f9:ffcf:bf9d
d0f1:f603:8e9:98c:3f5f:1f9:ffcf:bf9c
d0f1:f603:8e9:98c:3f5f:1f9:ffcf:bf98
d0f1:f603:8e9:6cb:803f:3e39:f134:3fbe
d0f1:f603:8e9:6cb:803f:3e39:f134:3fbd
d0f1:f603:8e9:6cb:803f:3e39:f134:3fbc
d0f1:f603:8e9:6cb:803f:3e39:f134:3fbb
